Packs/FeedOffice365/Integrations/FeedOffice365/FeedOffice365.py

```diff
@@ -168,6 +168,10 @@ def fetch_indicators(client: Client, indicator_type_lower: str, limit: int = -1)
                     "value": value,
                     "type": type_,
                     "rawJSON": raw_data,
+                    "fields": {
+                        "port": item.get('tcpPorts', ''),
```

Comment:
How is `'tcpPorts'` returned, as a list?

Reply:
String. take a look:
`[
  {
    "id": 31,
    "serviceArea": "SharePoint",
    "serviceAreaDisplayName": "SharePoint Online and OneDrive for Business",
    "urls": [
      "*.sharepoint.com"
    ],
    "ips": [
      "13.107.136.0/22",
      "40.108.128.0/17",
      "52.104.0.0/14",
      "104.146.128.0/17",
      "150.171.40.0/22",
      "2620:1ec:8f8::/46",
      "2620:1ec:908::/46",
      "2a01:111:f402::/48"
    ],
    "tcpPorts": "80,443",
    "expressRoute": true,
    "category": "Optimize",
    "required": true
  }`

Reply:
I think the `port` indicator field accepts lists so I think we need to `.split(',')` here.